BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -27,7 +27,7 @@ int update_Time(int row)
 
     }
  
- int subt = reload - current ;
+ int8_t subt = reload - current ;
```

Comment:
This function shouldn't be returning anything. Its function should be to write the new reload value to the CSV file like a real timer would update the CURRENT register. It should also call upon whatever function it was initialized to if the timer reaches 0.